### PR TITLE
Add py-oslo-i18n

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

**Comment by adamjstewart** [comment]
@spackbot commands

**Comment by adamjstewart** [comment]
@spackbot maintainers

File: var/spack/repos/builtin/packages/py-oslo-i18n/package.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    homepage = "https://docs.openstack.org/oslo.i18n"
+    pypi     = "oslo.i18n/oslo.i18n-5.0.1.tar.gz"
+
+    maintainers = ['haampie']
+
+    version('5.0.1', sha256='3484b71e30f75c437523302d1151c291caf4098928269ceec65ce535456e035b')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-pbr@2:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-oslo-i18n/package.py]:
```suggestion
    depends_on('py-pbr@2.0.0:2.0.999,2.1.1:', type=('build', 'run'))
```

Comment (haampie) [inline var/spack/repos/builtin/packages/py-oslo-i18n/package.py]:
```suggestion
    depends_on('py-pbr@2.0.0:2.0.999,2.1.1:', type='build')
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-oslo-i18n/package.py (hunk 1 above)
Comment (haampie) [reply var/spack/repos/builtin/packages/py-oslo-i18n/package.py]: haha, beat me to it

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**